tests/modules/propagate.c

```diff
@@ -390,6 +405,11 @@ int ValkeyModule_OnLoad(ValkeyModuleCtx *ctx, ValkeyModuleString **argv, int arg
                 "write",1,1,1) == VALKEYMODULE_ERR)
             return VALKEYMODULE_ERR;
 
+    if (ValkeyModule_CreateCommand(ctx,"propagate-test.obeyed",
+        propagateTestObeyed,
+        "",1,1,1) == VALKEYMODULE_ERR)
+    return VALKEYMODULE_ERR;
```

Comment:
Why return error here?

Reply:
Thanks - I missed this indentation.

Fixed it here: https://github.com/valkey-io/valkey/pull/1582/commits/93acad9f71e3ed04c656739ca96cfb7f852505cd